signer/signer_trust.go

```diff
@@ -85,3 +96,41 @@ func (trust *NotarySigner) GetKey(keyid string) data.PublicKey {
 	}
 	return data.NewPublicKey(data.KeyAlgorithm(publicKey.KeyInfo.Algorithm.Algorithm), publicKey.PublicKey)
 }
+
+// CheckHealth returns true if trust is healthy - that is if it can connect
+// to the trust server, and both the key management and signer services are
+// healthy
+func (trust *NotarySigner) CheckHealth(timeout time.Duration) error {
```

Comment:
Ah, ok, I originally thought that maybe if we provide a new constructor (since the struct itself doesn't say whether they are the same server) that had separate endpoints, checking both would make sense, but you're right, we can check both services later if we add that.